versions.mk

```diff
@@ -18,7 +18,7 @@ MODULE := github.com/NVIDIA/$(DRIVER_NAME)
 
 REGISTRY ?= nvcr.io/nvidia/cloud-native
 
-VERSION  ?= v25.3.0
+VERSION  ?= v25.3.0-rc.1
```

Comment:
In the Makefile, my assumption is that we want to keep the `v` prefix, and also our git tag strategy will continue to use the `v` prefix.

Reply:
https://github.com/NVIDIA/k8s-device-plugin/blob/f171c9269279c5021b286c9a7ca9a428bb673bea/hack/package-helm-charts.sh#L25